.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -16,28 +15,17 @@ body:
         Thanks for taking the time to fill out this bug report! Please include as much information as possible to help us fix the issue. 
 
         Tip: You can attach images or videos by dragging it into the text box.
-  - type: input
-    attributes:
-      label: Cody Version
-      description: Please specify the exact Cody extension version you're reporting for.
-    validations:
-      required: true
-  - type: textarea
+  - id: about
+    type: textarea
     attributes:
       label: IDE Information
       description: Trigger the action "About" (Shift Shift -> Actions), click on "Copy and Close" and paste the output here.
     validations:
       required: true
   - type: textarea
     attributes:
-      label: Describe the bug
-      description: Description of what the bug is. Please include steps on how to reproduce the behaviour.
-    validations:
-      required: true
-  - type: textarea
-    attributes:
-      label: Expected behavior
```

Comment:
too complicated. we received a lot of error with no details filled in anyway. a single required field should be enough